responsive.css

```diff
@@ -0,0 +1,210 @@
+
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/159893906-49ac512a-e422-4ca2-90b9-84cbdbca9646.png)

You are using only Flexbox.

Therefore:

- [ ] Please, ensure you are using CSS Grid, in addition to Flexbox.

That way, we will meet this requirement 💪